sensing/autoware_pointcloud_preprocessor/launch/distortion_corrector_node.launch.xml

```diff
@@ -1,5 +1,5 @@
 <launch>
-  <arg name="input/pointcloud" default="/sensing/lidar/top/mirror_cropped/pointcloud_ex"/>
+  <arg name="input/pointcloud" default="/sensing/lidar/top/pointcloud_raw_ex"/>
```

Comment:
By using the `/sensing/lidar/top/mirror_cropped/pointcloud_ex`, the azimuth will not update since it is in the `base_link` instead of the `sensor_frame`. (This is mentioned in the Readme)
However, I think I should switch it back to the original one. Thanks!

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/b156fb59cee6536173178bd15941e4f8d7fa0fe6